content-cache_rock/entrypoint.sh

```diff
@@ -5,18 +5,12 @@
 
 set -eu
 
-# https://pempek.net/articles/2013/07/08/bash-sh-as-template-engine/
-render_template() {
-    eval "echo \"$(cat "$1")\""
-}
-
 cp /srv/content-cache/files/nginx-logging-format.conf /etc/nginx/conf.d/nginx-logging-format.conf
 
 # https://bugs.launchpad.net/juju/+bug/1894782
 JUJU_UNIT=$(basename /var/lib/juju/tools/unit-* | sed -e 's/^unit-//' -e 's/-\([0-9]\+\)$/\/\1/')
 export JUJU_UNIT
 
-render_template /srv/content-cache/templates/nginx_cfg.tmpl > /etc/nginx/sites-available/default
+cat /srv/content-cache/templates/nginx_cfg.tmpl > /etc/nginx/sites-available/default
```

Comment:
Shouldn't this be managed by the charm to avoid using an entrypoint script ?

Reply:
See below

Reply:
I'm trying to understand why you need to copy the template file inside the rock image since it will be overridden by the charm afterwards. Is it to make sure that nginx in the container starts without complaining (in the case where the file would be absent) ?

Reply:
That is the idea.